engine/execution/testutil/fixtures.go

```diff
@@ -41,20 +39,6 @@ func CreateContractDeploymentTransaction(contractName string, contract string, a
 		AddAuthorizer(authorizer).
 		AddAuthorizer(chain.ServiceAddress())
 
-	// to synthetically generate event using Cadence code we would need a lot of
-	// copying, so its easier to just hardcode the json string
-	// TODO - extract parts of Cadence to make exporting events easy without interpreter
-
-	interpreterHash := runtime.CodeToHashValue(script)
-	hashElements := interpreterHash.Elements()
-
-	valueStrings := make([]string, len(hashElements))
-
-	for i, value := range hashElements {
-		uint8 := value.(interpreter.UInt8Value)
-		valueStrings[i] = fmt.Sprintf("{\"type\":\"UInt8\",\"value\":\"%d\"}", uint8)
-	}
-
```

Comment:
@m4ksio I removed this part and everything seems working fine, and I missing something here ?